tests/phpunit/tests/link/getAdjacentPost.php

```diff
@@ -350,4 +350,81 @@ public function filter_excluded_terms( $excluded_terms ) {
 		$excluded_terms[] = $this->exclude_term;
 		return $excluded_terms;
 	}
+
+	/**
+	 * @ticket 41131
+	 */
+	public function test_get_adjacent_post_cache() {
```

Comment:
This needs a little work:

* Each assertion needs a message to make it easier to distinguish between failures.
* Use `get_num_queries()` instead of the wpdb global
* For assertions based on the number of queries, use `get_num_queries() - $start_num_queries` as the actual value. This will make failures clearer (`2 is not the same as 1` is easier to understand than `596 is not the smae as 595`)

Reply:
Fixed.